bentoml/_internal/server/cli/dev_api_server.py

```diff
@@ -9,23 +9,33 @@
 
 from ...log import LOGGING_CONFIG
 from ...trace import ServiceContext
+from ...configuration.containers import DeploymentContainer
 
 
 @click.command()
 @click.argument("bento_identifier", type=click.STRING, required=False, default=".")
 @click.option("--bind", type=click.STRING, required=True)
-@click.option("--backlog", type=click.INT, default=2048)
 @click.option("--working-dir", required=False, type=click.Path(), default=None)
+@click.option("--backlog", type=click.INT, default=2048)
+@click.option(
+    "--prometheus-dir",
+    type=click.Path(exists=True),
```

Comment:
Don't we ensure the directory exists and create it now?

Reply:
Here could be multiple workers, if we do the init here, there would be a potential contest. Then we will have to introduce a file lock, and remember to clean it if an exception happens.